azure-pipelines.yml

```
@@ -68,6 +68,10 @@ stages:
             parameters:
               environment: 'stg'
               resource_key: 'appgw'
+          - template: devops-templates/terraform-plan-template.yaml
+            parameters:
+              environment: 'perftest'
```

this needs to be test
```suggestion
              environment: 'test'
```

---

this should be renamed as well:
https://github.com/hmcts/cnp-aks-rbac-pipelines/blob/master/terraform/perftest.tfvars
to test.tfvars

---

done